### PR TITLE
PRGZ Q330/3+L4C-3D to Centaur+LE-3Dlite MkII swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing Q330/3 (2366) datalogger and L4C-3D (2991) sensor
 - Replacing with Centaur (9889) and Le-3Dlite MkII (G-0555)
 - Using most recent Centaur firmware version 4.8.5
 - Streams affected loc 10 EH-ENZ

### CODE REVIEW

**Review by salichon — DISMISSED** [review]
ta!

**Review by mnaguit — CHANGES_REQUESTED** [review]
Sensor is `MKIII` (instead of `MKII`)

Thanks @Thomas-Benson

File: assets/sensors.csv [hunk 1]
```diff
@@ -565,6 +565,7 @@ Lennartz Electronic,LE-3DliteMkII,F-0381,13048,
 Lennartz Electronic,LE-3DliteMkII,G-0445,,
 Lennartz Electronic,LE-3DliteMkII,G-0524,,
 Lennartz Electronic,LE-3DliteMkII,G-0525,,
+Lennartz Electronic,LE-3DliteMkII,G-0555,,
```
[the remark this answers is not in the document]
Comment (Thomas-Benson) [reply assets/sensors.csv]: Fixed, sorry

File: install/sensors.csv [hunk 2]
```diff
@@ -680,6 +680,7 @@ Lennartz Electronic,LE-3DliteMkII,F-0379,HD61,10,0,,0,0,0,0,0,0,2016-07-18T03:06
 Lennartz Electronic,LE-3DliteMkII,F-0381,CTND,10,0,,0,0,0,0,0,0,2010-09-05T01:35:02Z,2010-09-29T23:00:00Z
 Lennartz Electronic,LE-3DliteMkII,F-0381,HD60,10,0,,0,0,0,0,0,0,2016-03-09T07:00:02Z,2019-05-07T01:32:36Z
 Lennartz Electronic,LE-3DliteMkII,G-0445,KQ03,10,0,,0,0,0,0,0,0,2008-06-20T00:00:02Z,2017-05-04T02:00:00Z
+Lennartz Electronic,LE-3DliteMkII,G-0555,PRGZ,10,0,,0,0,0,0,0,0,2023-06-26T20:28:00Z,9999-01-01T00:00:00Z
```
[the remark this answers is not in the document]
Comment (Thomas-Benson) [reply install/sensors.csv]: Also fixed

**Review by salichon — APPROVED** [review]
sweet

File: assets/sensors.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (mnaguit) [reply assets/sensors.csv]: Thanks @Thomas-Benson 👍

**Review by mnaguit — APPROVED** [review]

**Review by CBurton90 — APPROVED** [review]

**Pull request merged**